.github/workflows/system-tests.yml

```diff
@@ -96,80 +96,71 @@ jobs:
           sudo python ./tools/src/openssl_config_fix.py
         working-directory: neofs-testcases
 
-################################################################
-      - name: Get TAG for docker images
-        run: |
-          echo "CURRENT_TAG=$( make version | sed 's/^v//' )" >> $GITHUB_ENV
-        working-directory: neofs-s3-gw
-
-      - name: Build neofs-s3-gw docker image
+      - name: Build neofs-node
+        timeout-minutes: 5
         run: |
-          make
-        working-directory: neofs-s3-gw
+          make all
+        working-directory: neofs-node
 
       - name: Build neofs-s3-gw binaries
         run: |
-          make image
+          make all
         working-directory: neofs-s3-gw
 
-      - name: Add NeoFS S3-gw TAGs to s3-gw-test env config
-        run: |
-          sed -i -e 's/S3_GW_VERSION=_TAG_/S3_GW_VERSION=${{ env.CURRENT_TAG }}/' .github/testcases-env
-        working-directory: neofs-s3-gw
-
-      - name: Copy testcases-env file to .env for neofs-dev-env
-        run: |
-          cp .github/testcases-env ${GITHUB_WORKSPACE}/neofs-dev-env/.env
-        working-directory: neofs-s3-gw
-
-################################################################
+      - name: Download latest stable neofs-rest-gw
```

Comment:
Do we need to download REST at all for these tests?

Reply:
Removed